xmppserver/src/main/java/org/jivesoftware/openfire/XMPPServer.java

```diff
@@ -1754,4 +1757,17 @@ public void setRemoteSessionLocator(RemoteSessionLocator remoteSessionLocator) {
     public boolean isStarted() {
         return started;
     }
+
+    public void sendMessageToAdmins(final String message) {
```

Comment:
UpdateManager has similar code. We should unify that.

Reply:
You know what? I thought I had. My bad. https://github.com/igniterealtime/Openfire/pull/1332